src/grammar.js

```diff
@@ -7,23 +7,25 @@ const grammar = ohm.grammar(`
       = term*
 
     term
-      = id
-      | word
-      | value
+      = value
+      | hole
+      | id
       | variable
       | wildcard
-      | hole
-
-    id
-      = "#" alnum*
+      | word
+      | ws
 
     value
-      = keyword<"true">   -- true
-      | keyword<"false">  -- false
-      | keyword<"null">   -- null
+      = "true"   -- true
+      | "false"  -- false
+      | "null"   -- null
```

Comment:
I don't think we need the comments here anymore.

I like the cleanup, but are we losing anything in removing the distinction between
"things prefixed with symbols which clearly indicate that there is some special meaning"
vs
"alphanumeric works we just decided have special meaning and might be confusing that they parse as a value?"

For first time contributors, are they losing debugging information when their sentence goes wrong? Right now I don't think so, so this should be fine.

Reply:
> I don't think we need the comments here anymore.

The comments do dispatch within the semantics so we still need that (if I understand what ohm do).

---

> For first time contributors, are they losing debugging information when their sentence goes wrong? Right now I don't think so, so this should be fine.

We don't really have super descriptive error messages right now in the parser either, I think we can get them out of the parser generator using [comments](https://github.com/harc/ohm/blob/master/doc/syntax-reference.md#rule-descriptions)?

Reply:
oh dang, definitely would love comments

and misread -- as lua comment

Reply:
Was playing around with `ohm` and hit this https://github.com/harc/ohm/pull/218 🤷‍♀️